### ISSUE TITLE
Dall-e Prompts broken

### ISSUE THREAD

**Issue by nilopmsft** [nilopmsft/dnd-campaign-copilot-api#26]
Looks like the Dall-e prompts are broken again in that they are not being used for prompting dall-e beyond what is passed.  Thus creating an image that is somewhat relevant but not that great. Could be a class problem or completion formatting issue.

**Comment by nilopmsft** [comment]
Locale seems to be a good example of we see a prompt in our completion but for whatever reason we are not sending that to Dalle?

```
[2024-10-05T10:45:11.033Z] Locale Completion:
[2024-10-05T10:45:11.033Z] { "name": "Crystalfall Basin", "description": "Crystalfall Basin is a surreal and vibrant locale where the Aether Streams spill over into the physical realm in cascading energy waterfalls. The basin is a kaleidoscope of shifting colors and geometric patterns that glow with an otherworldly luminescence. It serves as a neutral ground where delegates from both the Luminar Confederacy and the Iron Dominion come to negotiate and study the phenomena. Adventurers often venture here to obtain Aether-infused crystals, rumored to hold immense power. The area is dotted with floating platforms and crystalline structures that defy gravity, creating an environment that feels both serene and charged with potential energy. Strange flora and fauna, native to both Aether and the physical world, thrive here, adding to the locale's mystique.", "type": "Mystical Landscape", "dallePrompt": "A surreal and vibrant landscape where energy streams spill like waterfalls into a colorful basin, glowing with otherworldly luminescence. Floating platforms and crystalline structures defy gravity, surrounded by strange flora and fauna. A meeting place for scholars and technocrats, this vivid scene is both serene and charged with mystical energy." }
[2024-10-05T10:45:11.036Z] Dalle Prompt:
[2024-10-05T10:45:11.036Z]  
```

**Comment by nilopmsft** [comment]
fixed with #34
